tools/api-tools/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.user
```

Comment:
This file shouldn't be needed, all these are covered in the top-level .gitignore.